libs/shared/cms/src/__generated__/gql.ts

```diff
@@ -11,8 +11,20 @@ import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/
+    "\n  query PurchaseWithDetailsOfferingContent(\n    $locale: String!\n    $stripePlanIds: [String]!\n  ) {\n    purchases(\n      filters: {\n        or: [\n          { stripePlanChoices: { stripePlanChoice: { in: $stripePlanIds } } }\n          {\n            offering: {\n              stripeLegacyPlans: { stripeLegacyPlan: { in: $stripePlanIds } }\n            }\n          }\n        ]\n      }\n      pagination: { limit: 500 }\n    ) {\n      stripePlanChoices {\n        stripePlanChoice\n      }\n      purchaseDetails {\n        details\n        productName\n        subtitle\n        webIcon\n        localizations(filters: { locale: { eq: $locale } }) {\n          details\n          productName\n          subtitle\n          webIcon\n        }\n      }\n      offering {\n        stripeProductId\n        stripeLegacyPlans(pagination: { limit: 200 }) {\n          stripeLegacyPlan\n        }\n        commonContent {\n          privacyNoticeUrl\n          privacyNoticeDownloadUrl\n          termsOfServiceUrl\n          termsOfServiceDownloadUrl\n          cancellationUrl\n          emailIcon\n          successActionButtonUrl\n          successActionButtonLabel\n          newsletterLabelTextCode\n          newsletterSlug\n          localizations(filters: { locale: { eq: $locale } }) {\n            privacyNoticeUrl\n            privacyNoticeDownloadUrl\n            termsOfServiceUrl\n            termsOfServiceDownloadUrl\n            cancellationUrl\n            emailIcon\n            successActionButtonUrl\n            successActionButtonLabel\n            newsletterLabelTextCode\n            newsletterSlug\n          }\n        }\n      }\n    }\n  }\n": typeof types.PurchaseWithDetailsOfferingContentDocument,
+    "\n  query ServicesWithCapabilities {\n    services(pagination: { limit: 500 }) {\n      oauthClientId\n      capabilities {\n        slug\n      }\n    }\n  }\n": typeof types.ServicesWithCapabilitiesDocument,
+};
```

Comment:
If this is needed, this should be created as a separate PR rather than in this one.